sycl/source/handler.cpp

```diff
@@ -404,6 +404,13 @@ void handler::processArg(void *Ptr, const detail::kernel_param_kind_t &Kind,
   }
 }
 
+// The argument can take up more space to store additional information about
+// MAccessRange, MMemoryRange, and MOffset added with addArgsForGlobalAccessor.
+// We use the worst-case estimate because the lifetime of the vector is short.
```

Comment:
Could you please clarify why `12` is considered to be worst-case estimate?
A kernel which takes 4 accessors should already have 16 arguments.

Reply:
Actually, you are right, 12 is wrong, I recheck the code, and the correct value is 13.
Now, why 13? because in `processArg` the kind_stream case introduces the maximum number of additional arguments.
The case adds additional 12 arguments to the currently processed argument, hence worst-case estimate is 12+1=13.
So I updated the code.

Reply:
@alexanderfle could you please add this justification as a comment?

Reply:
@alexbatashev, Done.